yt_dlp/extractor/picarto.py

```diff
@@ -10,7 +10,8 @@
 
 
 class PicartoIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www.)?picarto\.tv/(?P<id>[a-zA-Z0-9]+)'
+    IE_NAME = 'picarto:base'
```

Comment:
"base" could be misinterpreted to mean a baseclass
```suggestion
    IE_NAME = 'picarto'
```

Reply:
I've updated it.